api/eventsingress_api.md

```diff
@@ -0,0 +1,49 @@
+<!--:nav_order:2-->
+
+# Events Publishing API
+
+## Prerequisites
+
+1. Create a project and a workspace in the [`Adobe I/O Developer console`](https://www.adobe.io/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/projects-empty.md)
+2. Add the `I/O Management API` in your `Adobe I/O Developer console` project 
+3. [Generate a JWT token](https://www.adobe.io/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/credentials.md)
+4. Using [Adobe I/O Events Management API](provider_api.md) 
+   * create your own custom Adobe I/O Events provider
+   * create at least one event metadata associated with the above
+
+## Test Drive
+
+Once its metadata persisted in Adobe I/O Events (see above prerequisites)
```

Comment:
Once its metadata **is** persisted

Reply:
done https://github.com/AdobeDocs/adobeio-events/pull/77/commits/faec76e2a7e8a330287f0d6d21e534b8f71e071c